be/src/storage/vectorized/delta_writer.cpp

```diff
@@ -37,8 +37,9 @@ DeltaWriter::~DeltaWriter() {
     switch (_get_state()) {
     case kUninitialized:
     case kCommitted:
+    case kInitialized:
         break;
-    case kWriting:
+    case kPrepared:
```

Comment:
What does kWriting mean?

Reply:
> What does kWriting mean?

`kWriting` means  this delta_wirter has data written, and it will exec `rollback` if some error happened during transaction.